To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
+++ ../src-work/minecraft/net/minecraft/network/play/client/CPacketCustomPayload.java
@@ -47,7 +47,11 @@
     public void func_148840_b(PacketBuffer p_148840_1_) throws IOException
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
         p_148840_1_.func_180714_a(this.field_149562_a);
+        synchronized(this.field_149561_c) { //This may be access multiple times, from multiple threads, lets be safe.
+        this.field_149561_c.markReaderIndex();
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
+        this.field_149561_c.resetReaderIndex();
+        }
     }
To remove copyright: 1) open `blr.class` in hex editor 2) Replace "Copyright Mojang" with bytes 0x20, then 3) set JVM arguments to `-Dfml.ignorePatchDiscrepancies=true -Dfml.ignoreInvalidMinecraftCertificates=true`
     public void func_148833_a(INetHandlerPlayServer p_148833_1_)
